### PR TITLE
docs: chinese docs of developer guide overview

### PR DESCRIPTION
**Pull request by nicecui**

### CODE REVIEW

**Review by xtang — APPROVED** [review]

**Comment by xtang** [comment]
@nicecui thanks, and it looks good to me after resolving the conflicts.

File: docs/zh/developer-guide/overview.md [hunk 1]
```diff
@@ -1,65 +1,41 @@
...
-`GreptimeDB` consists of the following key components:
+`GreptimeDB` 由以下关键组件组成：
 
-- `Frontend` that exposes read and write service in various protocols, forwards requests to
-  `Datanode`.
-- `Datanode` is responsible for storing data to persistent storage such as local disk, S3.
-- `Meta` server that coordinates the operations between the `Frontend` and `Datanode`.
+- `Frontend`：通过多种协议提供读写服务，并将请求转发到 `Datanode`。
+- `Datanode`：负责数据的持久化存储，例如存储到本地磁盘、S3 和 Azure Blob Storage 等。
+- `Meta` 服务器：协调 `Frontend` 和 `Datanode` 之间的操作。
```
Comment (fengjiachun) [inline docs/zh/developer-guide/overview.md:9]:
```suggestion
- `Metasrv` 元数据存储和分布式调度：协调 `Frontend` 和 `Datanode` 之间的操作。
```

File: docs/developer-guide/overview.md [hunk 2]
```diff
@@ -9,7 +9,7 @@
 - `Datanode` is responsible for storing data to persistent storage such as local disk or object storage in the cloud such as AWS S3, Azure Blob Storage etc.
 - `Meta` server that coordinates the operations between the `Frontend` and `Datanode`.
```
Comment (fengjiachun) [inline docs/developer-guide/overview.md:10]:
```suggestion
- `Metasrv` server that coordinates the operations between the `Frontend` and `Datanode`.
```

File: docs/zh/developer-guide/overview.md [hunk 3]
```diff
@@ -1,65 +1,41 @@
...
+- `table engine` 是 `GreptimeDB` 处理不同 `table` 类型的 SQL 操作的组件。`Mito` 是默认的 `table engine`。它使用基于 LSM-Tree 的 `storage engine` 作为其存储后端。
 
-## How it works
+## 工作原理
 
-![Interactions between components](../../public/how-it-works.png)
+![Interactions between components](/how-it-works.png)
 
-Before diving into each component, let's take a high level view of how the database works.
+在深入了解每个组件之前，让我们先从高层次上了解一下 GreptimeDB 数据库的工作原理。
```
Comment (fengjiachun) [inline docs/zh/developer-guide/overview.md:30]:
```suggestion
在深入了解每个组件之前，让我们先从高层次上了解一下 GreptimeDB 的工作原理。
```

File: docs/zh/developer-guide/overview.md [hunk 4]
```diff
@@ -1,65 +1,41 @@
...
-  `metasrv`.
-- The `frontend` periodically fetches and caches the metadata from `metasrv`, thus it knows which
-  `datanode`, or more precisely, the region a request should be sent to. A request may be split and
-  sent to multiple `regions`, if its contents need to be stored in different `regions`.
-- When `datanode` receives the request, it writes the data to the table, and then sends response
-  back to the `frontend`. Writing to the table will then write to the underlying storage engine,
-  which will eventually put the data to persistent device.
-- Once `frontend` has received all responses from the target `datanode`s, it then sends the result
-  back to the user.
+- 用户可以通过各种协议与数据库交互，例如使用 `InfluxDB` 的行协议插入数据，然后使用 SQL 或 PromQL 探索数据。`frontend` 是用户或客户端连接和操作数据库的组件，因此在其后面隐藏了 `datanode` 和 `metasrv`。
```
Comment (fengjiachun) [inline docs/zh/developer-guide/overview.md:32]:
```suggestion
- 用户可以通过各种协议与数据库交互，例如使用 `InfluxDB line Protocol` 插入数据，然后使用 SQL 或 PromQL 查询数据。`frontend` 是用户或客户端连接和操作数据库的组件，因此在其后面隐藏了 `datanode` 和 `metasrv`。
```

File: docs/zh/summary-i18n.yml [hunk 5]
```diff
@@ -7,12 +7,12 @@ Query-Data: 读取数据
...
 Benchmarks: 基准
 How-To: 指南
 Biweekly-Reports: Biweekly-Reports
 GreptimeCloud: 云服务
-Integrations: 集成
\ No newline at end of file
+Integrations: 集成
+Frontend: Frontend
+Datanode: Datanode
+Meta: Meta
```
Comment (fengjiachun) [inline docs/zh/summary-i18n.yml:18]:
```suggestion
Metasrv: Metasrv
```

Comment (fengjiachun) [reply docs/zh/summary-i18n.yml:18]: Better to add an empty line at the end of the file

**Review by fengjiachun — APPROVED** [review]

**Pull request merged**